### PR TITLE
Issue 1728/ Add third button to modus modal

### PR DESCRIPTION
**Pull request by pchaudh18**
## Description

Adding third button in modus modal
References #1728

## Type of change

- [Y] Bug fix (non-breaking change which fixes an issue)
- [Y] New feature (non-breaking change which adds functionality)
- [Y] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [Y] Documentation update

## How Has This Been Tested?

modus-modal.spec.ts
modus-modal.e2e.ts

## Checklist

- [Y ] My code follows the style guidelines of this project
- [Y ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [Y ] My changes generate no new warnings
- [Y] I have added tests that prove my fix is effective or that my feature works
- [Y ] New and existing unit tests pass locally with my changes
- [Y] Any dependent changes have been merged and published in downstream modules
- [Y ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by msankaran0712 — CHANGES_REQUESTED** [review]
Missing updates on Storybook pages

File: stencil-workspace/src/components/modus-modal/modus-modal.scss [hunk 1]
```diff
@@ -69,7 +69,7 @@
       justify-content: flex-end;
       padding: 0 $rem-16px;
 
-      modus-button:first-of-type {
+      modus-button:nth-of-type(2) {
```
Comment (msankaran0712) [inline stencil-workspace/src/components/modus-modal/modus-modal.scss:72]: is this tested?

File: stencil-workspace/src/components/modus-modal/readme.md [hunk 2]
```diff
@@ -7,18 +7,13 @@
 
 ## Properties
 
-| Property                   | Attribute                     | Description                                                                                                                          | Type                    | Default     |
```
Comment (msankaran0712) [inline stencil-workspace/src/components/modus-modal/readme.md:10]: Please copy this file for updating the properties section on the Storybook.

File: stencil-workspace/src/components/modus-modal/modus-modal.models.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+export interface ModusModalModels {
+  /** (optional) The modal's  button text. */
+  text?: string;
+  /** (optional) The modal's  button aria-label. */
+  ariaLabel?: string;
+  /** (optional) Disable  button. */
+  disabled?: boolean;
+}
+
+export interface ModusModalButtons {
```
[the remark this answers is not in the document]
Comment (pchaudh18) [reply stencil-workspace/src/components/modus-modal/modus-modal.models.tsx:10]: updated with one change as one file created

File: stencil-workspace/src/components/modus-modal/modus-modal.scss (hunk 1 above)
Comment (pchaudh18) [reply stencil-workspace/src/components/modus-modal/modus-modal.scss:72]: yes for space between the buttons

File: stencil-workspace/src/components/modus-modal/readme.md (hunk 2 above)
Comment (pchaudh18) [reply stencil-workspace/src/components/modus-modal/readme.md:10]: updated the storybook modus file

**Comment by msankaran0712** [comment]
@trimble-oss/mwc-maintainers @ryan-henness-trimble @coliff @cjwinsor, Please note that the PR will be a breaking change for Modal component

**Comment by msankaran0712** [comment]
A new PR https://github.com/trimble-oss/modus-web-components/pull/1786 is available hence closing this one

**Pull request closed without merging**